### PR TITLE
scikit-learn classifier wrapper

### PR DESCRIPTION
**Pull request by jdwittenauer**
Resolves issue #149

Some quick notes/observations:

- After a few design iterations, I opted to use a constructor that accepts a pre-defined (but not yet compiled) Keras model.  I'm reasonably sure this is the most sensible approach, but not 100% sure.  The advantage is that one can still build model definitions using the existing APIs, which are very fluid and expressive.  The disadvantage is that it's not possible to build a model using all default parameters, as one can do with the other scikit-learn estimators (I'm not sure this even makes sense for a neural net though).
- In order to allow meta-estimators such as ensembles to make copies of the base estimator, I'm making a copy of the model definition in the fit method before compilation.  I don't think this will cause any problems, but I'm not positive.  I tested it on my "Otto Group" Kaggle script and was able to build an ensemble using the wrapper class and scikit-learn's bagging classifier.
- I only created a classification wrapper.  It would probably make sense to pull most of this out into an abstract base class and create derived classes for classification and regression instead.  I'll leave that to a future enhancement though.

### CODE REVIEW

**Comment by fchollet** [comment]
Looking good! Do you have any tests for this?

Agreed that a regression wrapper would also be a nice addition, and that it would probably make sense to have both wrappers inherit from a same class.

**Comment by jdwittenauer** [comment]
Not in a state that I can easily add to the project.  I can write up a test script though similar to what's in the "tests" folder already.

**Comment by fchollet** [comment]
A very test script that runs the different methods of the wrapper on a toy example would be neat : )

**Comment by jdwittenauer** [comment]
Added a new commit with the test script.

**Comment by fchollet** [comment]
Cool, thanks! I'm merging this.
